frontend/src/app/containers/ExperimentPage/tests.js

```diff
@@ -119,7 +121,7 @@ describe('app/containers/ExperimentPage:ExperimentDetail', () => {
       setPageTitleL10N: sinon.spy()
     };
 
-    subject = shallow(<ExperimentDetail {...props} />);
+    subject = mount(<ExperimentDetail {...props} />);
```

Comment:
Switching from shallow rendering to full component mount lets the ExperimentPage tests pass without breaking them up to work with individual sub-components. Figured this was a safer way to verify I hadn't broken anything